cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedCall.qll

```diff
@@ -87,7 +93,8 @@ abstract class TranslatedCall extends TranslatedExpr {
     )
   }
 
-  override Instruction getInstructionSuccessor(InstructionTag tag, EdgeKind kind) {
+  override Instruction getInstructionSuccessorInternal(InstructionTag tag, EdgeKind kind) {
+    kind instanceof GotoEdge and
```

Comment:
Is this line needed given that we're restricting the `EdgeKind` on the line below?